sample/playableGame.py

```diff
@@ -0,0 +1,52 @@
+import sample.board as board
+board = board.Board()
+
+def get_next_sub_board():
+    if board.next_move == (-1,-1):
+        #get user entered subX,subY
+        print("You can choose which sub_board you want to play in next.")
+        nextSub = input("Enter the row and column separated by a comma : ")
+        nextSub = nextSub.split(',')
+        (subX,subY) = tuple((int(str) for str in nextSub))
```

Comment:
I think changing the contents of a variable like this is not a good idea, since it becomes different things while having the same name. You could change this to
```Python
        raw_input = input("Enter the row and column separated by a comma : ")
        (subX,subY) = tuple((int(str) for str in raw_input.split(',')))